content.ts

```diff
@@ -52,6 +52,14 @@ export type EditContentOpts = {
     content?: string;
 }
 
+export type ContentListResult = {
+    title: string;
+    latestVersion: number;
+    id: string;
+    Content(): Promise<Content>;
+}
+
+export type ContentListRequestResult = { nextPage: () => Promise<ContentListRequestResult>, contents: ContentListResult[] };
```

Comment:
Mind if we change these to `ContentListItem` and `ContentListResult` respectively? Then it will be consistent with the documents list API.